packages/@react-aria/toast/src/useToast.ts

```diff
@@ -32,8 +31,10 @@ interface ToastAria {
 export function useToast(props: AriaToastProps): ToastAria {
   let {
     id,
+    idKey,
```

Comment:
small nitpick, feel free to push back
Feel like `idKey` is a bit unintuitive, maybe `toastKey`?